Taudem5PCVS2010/FlowdirConditioning/targetver.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
This file appears to have been automatically generated. I prefer not to have it. See if we can configure project without it.